extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -135,15 +136,21 @@ export class Bind {
     /** @const @private {!../../../src/service/viewer-impl.Viewer} */
     this.viewer_ = viewerForDoc(this.ampdoc);
 
-    /**
+    const bodyReadyPromise = (opt_win)
```

Comment:
`whenBodyAvailable` does, but `waitForBodyPromise` doesn't. Changed it to use resolve value but not sure it's cleaner.